script/smt-report

```diff
@@ -495,64 +432,22 @@ $LOG = SMT::Utils::openLog($logfile);
 # calculate best report
 my $cfg = SMT::Utils::getSMTConfig();
 
-if ( $cfg->val('NU', 'ApiType', 'NCC') eq 'NCC')
+my $api = $cfg->val('NU', 'ApiType', 'NCC');
+if ( $api eq 'NCC')
 {
-    printLog($LOG, $vblevel, LOG_ERROR, __("NCC is not supported in this version of SMT"));
+    printLog($LOG, $vblevel, LOG_ERROR, __("NCC is not supported by this version of SMT"));
     exit 1;
 }
-
-# sync current data from SCC
-if(!$nosccsync)
+if ( $api eq 'SCC' && $nccreport)
```

Comment:
remove the whole section. It does not make sense to add an option which only return an error if you use it.